### PR TITLE
Add news cli to winget pkgs

### PR DESCRIPTION
**Pull request by Atticus64**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109548&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm seeing this:
image
image
image

Bypassing Defender results I see this - It's missing VC Redist:
image

@stephengillie could probably make a waiver for this (but you should also include the VC Redist in the manifest so that you won't run into the https://github.com/microsoft/winget-pkgs/labels/Validation-Executable-Error issue anymore once dependency support is out of experimental features)

**Review by Trenly — COMMENTED** [review]
image

**Comment by stephengillie** [comment]
> I'm seeing this:

I couldn't replicate the scan output: 
```
4:58:25 PM: Detected 0 Defender (with signature version 1.391.1206.0) errors: 
4:58:25 PM: 
```

> 
> Bypassing Defender results I see this - It's missing VC Redist: image

This package installed normally in a Windows 11 VM:
```
PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Links\news.exe
CLI to watch news in the terminal Γëí╞Æ├ë├│ developed by Jonathan

Usage: news [OPTIONS] [COMMAND]

Commands:
  list   list latest news of a lang
  today  Check the latest news
  help   Print this message or the help of the given subcommand(s)

Options:
...
```

> @stephengillie could probably make a waiver for this (but you should also include the VC Redist in the manifest so that you won't run into the [ Validation-Executable-Error](https://github.com/microsoft/winget-pkgs/labels/Validation-Executable-Error) issue anymore once dependency support is out of experimental features)

File: manifests/a/Atticus64/news/1.3.0/Atticus64.news.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  Custom: /norestart
+Commands:
+- news
+Protocols:
+- http
+- https
+FileExtensions:
+- toml
+ReleaseDate: 2023-06-11
+Installers:
```
Comment (stephengillie) [inline manifests/a/Atticus64/news/1.3.0/Atticus64.news.installer.yaml:20]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Review by stephengillie — APPROVED** [review]

**Pull request merged**